source/tyr/tyr/resources.py

```diff
@@ -882,72 +882,34 @@ def put(self, version=0, id=None, name=None):
         )
 
         parser.add_argument(
-            "asgard_language",
+            "language",
             type=OptionValue(
                 [
-                    'bulgarian',
-                    'catalan',
-                    'czech',
-                    'danish',
-                    'german',
-                    'greek',
-                    'english_gb',
-                    'english_pirate',
-                    'english_us',
-                    'spanish',
-                    'estonian',
-                    'finnish',
+                    'dutch',
+                    'english',
                     'french',
+                    'german',
                     'hindi',
-                    'hungarian',
                     'italian',
                     'japanese',
-                    'bokmal',
-                    'dutch',
-                    'polish',
-                    'portuguese_br',
-                    'portuguese_pt',
-                    'romanian',
+                    'portuguese',
                     'russian',
-                    'slovak',
-                    'slovenian',
-                    'swedish',
-                    'turkish',
-                    'ukrainian',
+                    'spanish',
                 ]
             ),
-            help='Select a specific language for Asgard guidance instruction.\n'
+            help='Select a specific language for street network instructions.\n'
             'list available:\n'
-            '- bulgarian = bg-BG\n'
-            '- catalan = ca-ES\n'
-            '- czech = cs-CZ\n'
-            '- danish = da-DK\n'
-            '- german = de-DE\n'
-            '- greek = el-GR\n'
+            '- dutch = nl-NL\n'
             '- english_gb = en-GB\n'
```

Comment:
Should this be edited to remove the variant?

```suggestion
            '- english = en-GB\n'
```

Reply:
:+1: